distribution/docs/src/main/jdocs/content/_endpoints/cometd-endpoint-contents.adoc

```diff
@@ -0,0 +1,541 @@
+:title: CometD Endpoint
+:type: endpoint
+:status: published
+:filename: cometd-endpoint-contents.adoc
+:link: _cometd_endpoint
+:projectpath: {adoc-include}
+:summary: Enables asynchronous search capabilities.
+:implements: https://docs.cometd.org/current/reference/[CometD].
+
+The http://cometd.org[CometD] endpoint enables asychronous search capabilties.
+The CometD protocol is used to execute searches, retrieve results, and receive notifications.
+
+For an example of using CometD within a webapp see: `distribution/sdk/sample-cometd/`
+
+===== Installing CometD Endpoint
+
+The CometD Endpoint is installed by default with a standard installation in the ${ddf-ui} application.
+
+===== Configuring CometD Endpoint
+
+Configure the CometD endpoint from the ${admin-console}:
+
+. Navigate to the *${admin-console}*.
+. Select *${ddf-ui}* application.
+. Select *Configuration* tab.
+
+include::{adoc-include}/_tables/org.codice.ddf.ui.search.standard.endpoint-table-contents.adoc[]
```

Comment:
Saw this error here: `Unresolved directive in content/_endpoints/cometd-endpoint-contents.adoc - include::/Users/rlarsen/Builds/DDF/ddf/distribution/docs/target/doc-contents/content/_tables/org.codice.ddf.ui.search.standard.endpoint-table-contents.adoc[]`